js/client-chat.js

```diff
@@ -747,8 +747,13 @@
 				var targets = target.split(',');
 				var formatTargeting = false;
 				var formats = {};
+				var gens = {};
 				for (var i = 1, len = targets.length; i < len; i++) {
-					formats[toId(targets[i])] = 1;
+					if (targets[i].length === 4 && targets[i].indexOf('gen') === 0) {
```

Comment:
For the record, `string.substr(0, fragment.length) === fragment` is the usual way to test if a string starts with a fragment in basically every programming language ever.